docs/getting-started/contributors.md

```diff
@@ -142,7 +152,29 @@ default app configs.
 You can learn more about the local configuration in
 [Static Configuration in Backstage](../conf/) section.
 
-## Writing changesets
+## Creating Changesets
+
+We use [changesets](https://github.com/atlassian/changesets) to help us prepare releases. They help us make sure that every package affected by a change gets a proper version number and an entry in its `CHANGELOG.md`. To make the process of generating releases easy, it helps when contributors include changesets with their pull requests.
+
+### When to use a changeset?
+
+Any time a patch, minor, or major change aligning to [Semantic Versioning](https://semver.org) is made to any published package in `packages/` or `plugins/`, a changeset should be used. It helps to align your change to the [Backstage package versioning policy](https://backstage.io/docs/overview/versioning-policy#package-versioning-policy) for the package you are changing, for example, when to provide additional clarity on deprecation or impacting changes which will then be included into CHANGELOGs.
+
+In general, changesets are only needed for changes to packages within `packages/` or `plugins/` directories, and only for the packages that are not marked as `private`. Changesets are also not needed for changes that do not affect the published version of each package, for example changes to tests or in-line source code comments.
+
+Changesets **are** needed for new packages, as that is what triggers the package to be part of the next release.
```

Comment:
i feel like somewhere in here we should note if you update the README we should get a changeset. i myself have questioned that before, and i think the prevailing feedback is usually we want any README update published to Npm's web site and thus a changeset is preferred?

Reply:
Yeah, I agree but it didn't get in as I'm not sure how to word it within what already exists, if you have some suggestions I'm happy to add them!